src/explore-education-statistics-common/src/components/form/FormSelect.tsx

```diff
@@ -65,24 +67,30 @@ const FormSelect = ({
   return (
     <>
       <FormSelectWrapper inline={inline}>
-        <FormLabel
-          className={classNames({ 'govuk-!-margin-right-2': inline })}
-          id={id}
-          label={label}
-          hideLabel={hideLabel}
-        />
+        <div
+          className={classNames({
+            'dfe-flex dfe-justify-content--space-between': inlineHint,
+          })}
+        >
```

Comment:
Could we make this into a conditional container so we don't get the extra div in the markup when this shouldn't be there?

We could also write a snapshot test to cover this

Reply:
Done